src/backend/joanie/tests/core/test_utils_webhooks_synchronize_course_runs.py

```diff
@@ -145,7 +145,7 @@ def test_utils_synchronize_course_runs_failure(self, mock_error):
             rsp = rsps.post(
                 re.compile("http://richie.education/webhook"),
                 body="{}",
-                status=random.choice(["404", "500", "301"]),
+                status=random.choice([404, 301]),
```

Comment:
```suggestion
                status=random.choice([404, 500, 301]),
```

Reply:
I have to take the 500 out as it makes it reach the max retries

Reply:
So add 502 ? I just want to ensure this test is working when server errors are returned.